### PR TITLE
Wording tweaks to the Profiler and Queries pages

### PR DESCRIPTION
**Pull request by potocnyj**
Fix some language inconsistencies in the documentation for the Profiler and the Queries page. In addition, add some cross links to relevant content elsewhere in the docs, and provide a picture for the Host Breakdown section on the Queries page.

### CODE REVIEW

**Comment by potocnyj** [comment]
@Preetam or @fortxun thoughts here? Ideally I'd like to remove some of the duplicated content between these two pages, but I figured I'd start small

File: content/how-to-use-vividcortex/profiler.md [hunk 1]
```diff
@@ -29,17 +29,17 @@ The main area of the UI is very information-dense, so here is a deeper
 explanation of each item you might see, depending on what you've selected in the
 options panel at the top:
 
-* **Notifications** (the bullhorn icon) is the number of warnings/notifications we've detected for this family of queries. Navigate to the query details for more information.
+* **Notifications** (the bell icon) is the number of errors/warnings detected for this family of queries. Navigate to the [query details](/how-to-use-vividcortex/queries/#viewing-query-notifications) for more information.
```
Comment (Preetam) [inline content/how-to-use-vividcortex/profiler.md]: Nitpick: I'd write `errors or warnings`.

File: content/how-to-use-vividcortex/profiler.md [hunk 2]
```diff
@@ -29,17 +29,17 @@ The main area of the UI is very information-dense, so here is a deeper
...
 * **Slow Queries** is the number of times the query's execution time was longer than the `long_query_time` threshold, which defaults to 10 seconds. It usually does not correspond exactly to queries in the slow query log, because queries may be logged to the slow query log for a number of other reasons as well as just exceeding the threshold.
 * **CPU Time**, **IO Ops**, and other columns are computed estimates, not exact measures. See **Computed Columns** at the bottom of this page.
 
 ## Compare
 
-It is a quick way to answer the question “what changed?” when a server starts to behave differently, or to look at before-and-after behavior after a change to an application or an event such as a product launch. The image below shows a one-hour view of the profiler compared to an hour prior.
+This view provides a quick way to answer the question “what changed?” when a server starts to behave differently, or to look at before-and-after behavior of your database workload after a change to an application or an event such as a product launch. The image below shows a one-hour view of the profiler compared to an hour prior.
```
Comment (Preetam) [inline content/how-to-use-vividcortex/profiler.md]: `“what changed?”` -> `"what changed?"` for consistency. You could change them all to curly quotes; just keep it consistent.

File: content/how-to-use-vividcortex/profiler.md (hunk 1 above)
Comment (potocnyj) [inline content/how-to-use-vividcortex/profiler.md]: Notifications can be a mix of both errors and warnings; we can say `errors and/or warnings` but just saying `errors or warnings` isn't correct english

File: content/how-to-use-vividcortex/queries.md [hunk 3]
```diff
@@ -20,18 +20,18 @@ The top navigation bar has filters and options for changing what is displayed in
...
 ## The Main Table
 
 The main area of the UI shows a high level overview of each query returned from our APIs.
 
-* **Notifications** (the bullhorn icon) is the number of notifications (errors/warnings) we've detected for this query family. More information about notifications is below.
+* **Notifications** (the bell icon) is the number of errors/warnings detected for this query family. See the [Viewing Query Notifications](#viewing-query-notifications) section below for more information.
 * **Time** shows the query family text, as well as the sum of response times for the query over the selected time span (sparkline).
 * **First Seen** shows the date the query family was first seen by the application.
-* **Last Seen** shows the date the query family was last active.
+* **Last Seen** shows the approximate date the query family was last active.
```
Comment (Preetam) [inline content/how-to-use-vividcortex/queries.md]: Date or time?

File: content/how-to-use-vividcortex/queries.md [hunk 4]
```diff
@@ -94,4 +94,6 @@ As a result, VividCortex is unable to show you what the warnings were. We can on
 
 ## Host Breakdown
 
-The query's details page shows the hosts the query has been associated with during the time selected.
\ No newline at end of file
+The query's details page shows the hosts the query has been associated with during the time selected, along with the portion of the total time each host contributed toward what was seen. You can also see the breakdown of queries across hosts by count.
```
Comment (Preetam) [inline content/how-to-use-vividcortex/queries.md]: Don't we refer to that page as the "Query Details" page?
E.g. https://www.vividcortex.com/product-updates/updated-query-details-page-and-more

**Comment by Preetam** [comment]
LGTM besides comments 👍.

**Pull request merged**